src/wiki/getting-started/installing-java.md

```diff
@@ -9,24 +9,70 @@ eleventyNavigation:
 
-If you aren't sure which version you will need, **it is recommended to just install both.**
+If you would like to learn more about OpenJDK and the vendors mentioned in this page, here are their websites:
 
-**NOTE:** You can use any OpenJDK build you want. [Azul Zulu](https://www.azul.com/downloads/?package=jre#download-openjdk) is a popular alternative to Temurin, and will work just as well.
+- [Adoptium Temurin](https://adoptium.net/)
+- [Microsoft OpenJDK](https://www.microsoft.com/openjdk)
+- [Azul OpenJDK](https://www.azul.com/downloads/)
+- [OpenJDK source distribution](https://openjdk.org/)
 
-You can also use Oracle Java, but openjdk is better here. Also, to download Oracle Java 9+ you need an Oracle account which is not a good thing.
+Otherwise, continue reading for quick download links and a guide for which one to choose.
 
-## Java on linux
+## Installing Java on Windows
+
+**First, check your CPU architecture.** Here is a method that works on all Windows versions:
+
+1. Click or tap on the **Start Menu**
+2. Search **Command Prompt** or **Terminal** and click on it
+3. Type in `echo %PROCESSOR_ARCHITECTURE%` and press enter
+4. Read the result.
+   - If it says `AMD64`, you have a 64-bit **x86-64** CPU. This is sometimes called **x64** or **amd64**.
+   - If it says `X86`, you have a 32-bit **x86** CPU. This is sometimes called **x32** or **x86-32**.
+   - If it says `ARM64`, you have a 64-bit **ARM** CPU. This is sometimes called **aarch64** or **ARM64**.
+
+Then, download the appropriate Java:
+
+| Minecraft | CPU type | Download page | Viable alternatives |
+|:---:|:---:|---|---|
+| Minecraft **1.17** or above | x86-64 | [Microsoft OpenJDK 17 for Windows x64, `.msi` installer](https://aka.ms/download-jdk/microsoft-jdk-17-windows-x64.msi) | Azul, Coretto, Temurin, GraalVM |
+|  | x86 | [Temurin OpenJDK 17 for Windows x32, `.msi` installer](https://adoptium.net/temurin/releases/?release=17) | Azul, Coretto |
+|  | aarch64 | [Microsoft OpenJDK 17 for Windows aarch64, `.msi` installer](https://aka.ms/download-jdk/microsoft-jdk-17-windows-aarch64.msi) | Azul |
+| Minecraft **1.16** or below | x86-64 | [Temurin OpenJDK 8 for Windows x64, `.msi` installer](https://adoptium.net/temurin/releases/?release=8) | Azul, Coretto |
+|  | x86 | [Temurin OpenJDK 8 for Windows x64, `.msi` installer](https://adoptium.net/temurin/releases/?release=8) | Azul, Coretto |
+|  | aarch64 | Not available :( |  |
+
+**Tip:** If you are on Windows 11, you can automatically install 8 and 17 by running `winget install Microsoft.OpenJDK.17; winget install EclipseAdoptium.Temurin.8.JDK` in the Terminal app.
```

Comment:
This works on 10 as well. Anyone with [this](https://apps.microsoft.com/detail/app-installer/9NBLGGH4NNS1) installed (which ships with Win10/11 by default) can run the command.

Reply:
Is there a source that confirms that this has now been included with Win10? Winget was originally made for Windows 10, but it was only included by default on Windows 11 at first.

Reply:
its part of the microsoft store package and newer versions of windows 10 should ship it by default afaik.